sdk/core/core/inc/az_http_transport.h

```diff
@@ -89,24 +89,75 @@ az_http_request_get_header(_az_http_request const* request, int32_t index, az_pa
  * call this function like below to get only the http method and ignore getting url and body.
  *   `az_http_request_get_parts(request, &method, NULL, NULL)`
  *
- * This function is expected to be used by transport layer only.
+ * @remarks This function is expected to be used by transport layer only.
  *
- * @param request HTTP request to get parts from.
- * @param[out] out_method __[nullable]__ Pointer to write HTTP method to. Use `NULL` to ignore
+ * @param[in] request HTTP request to get parts from.
+ * @param[out] out_method __[nullable]__ Pointer to source HTTP method to. Use `NULL` to ignore
```

Comment:
Same here/elsewhere. The docs don't read well now that we have changed the parameter name.